### PR TITLE
Pre approval form and perform [2/2]

### PR DESCRIPTION
**Pull request by KrekoG**
### Added
- Pre-approval form and backend to perform (#160613620)

[Delivers [#160613620](https://www.pivotaltracker.com/story/show/160613620)]

### CODE REVIEW

File: resources/views/customer-intelligence/pre-approval-form.blade.php [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+                                onClose: () => {
+                                    redirectUser();
+                                }
+                            });
+                            break;
+                        case 'blue':
+                            swal({
+                                type: 'info',
+                                title: '',
+                                text: 'Merchant Liable response popup.', // TODO: consult with Sales to get wording. GK 2018-09-24
```
Comment (KrekoG) [inline resources/views/customer-intelligence/pre-approval-form.blade.php]: :hand: This PR is blocked by sales wording, but the technical review can still be carried out, as we are talking about purely text changes on front-end.

**Comment by KrekoG** [comment]
@wnowicki @EvanBarbour3 @00eadeyemi Ready for review

File: tests/unit/Services/CustomerIntelligenceServiceTest.php [hunk 2]
```diff
@@ -205,4 +205,65 @@ public function testPerformLeadScore()
...
+                        'town' => 'Altrincham',
+                        'postcode' => 'WA14 2DW',
+                    ],
+                    [
+                        'building_name' => 'WeWork No. 1 Spinningfields',
+                        'street' => 'Quay St',
+                        'town' => 'Manchester',
+                        'postcode' => 'M3 3JE',
+                    ],
+                ]
```
Comment (wnowicki) [inline tests/unit/Services/CustomerIntelligenceServiceTest.php]: - CS `],`

File: tests/functional/CustomerIntelligenceCest.php [hunk 3]
```diff
@@ -184,4 +184,109 @@ public function testPerformLeadScore(FunctionalTester $I)
...
+            'marketing'=> '1',
+            'addresses' => [
+                [
+                    'building_number' => '11',
+                    'street' => 'SUNNY STREET',
+                    'town' => 'TEST TOWN',
+                    'postcode' => 'X9 9AA',
+                    'moved_in' => '1995-02-01',
+                    'residential_status' => '4'
+                ]
```
Comment (wnowicki) [inline tests/functional/CustomerIntelligenceCest.php]: - CS `]`

File: tests/functional/CustomerIntelligenceCest.php [hunk 4]
```diff
@@ -184,4 +184,109 @@ public function testPerformLeadScore(FunctionalTester $I)
...
+            ]
```
Comment (wnowicki) [inline tests/functional/CustomerIntelligenceCest.php]: - CS `],`

**Pull request merged**